### PR TITLE
Add web app manifest file

### PR DESCRIPTION
**Pull request by chrisdwheatley**
I've added a basic manifest file with possible titles, colors etc... (happy to amend where necessary).

Is there a larger version of the images you used for the favicon @housseindjirdeh? That way I can add the icons and the manifest file will be ready for a2hs.

### CODE REVIEW

File: site/assets/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "HNPWA",
```
Comment (sindresorhus) [inline site/assets/manifest.json]: `Hacker News PWA`

File: site/assets/manifest.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "short_name": "HNPWA",
```
Comment (sindresorhus) [inline site/assets/manifest.json]: `HN PWA`

**Comment by chrisdwheatley** [comment]
Page title is currently 'HNPWA - Hacker News readers as Progressive Web Apps' want me to amend at all while I'm changing 'HNPWA'?

**Comment by housseindjirdeh** [comment]
@chrisdwheatley thanks for this! 🎉 🎉 🎉

I used [favicon.io](https://favicon.io/index.html) to generate the favicon. My apologies I should have generated the PNG as well earlier. [Here's a 512px .png](https://cloud.githubusercontent.com/assets/12476932/25675736/8c64eac6-300d-11e7-8489-c8a8ac7fee6b.png).

Feel free to play around with favicon.io to change up the favicon/image if you wish, here's a snapshot of the settings I used:

image

**Comment by housseindjirdeh** [comment]
As for the page title, I don't mind `Hacker News PWA` or the way it is currently, either one is good with me 👍

**Comment by addyosmani** [comment]
LGTM once we land the 512px icon.

**Comment by chrisdwheatley** [comment]
Have added 512px icon and reference to it in the manifest file.

Also optimised images with imageoptim-cli, amended page title and gitignore'd `.DS_Store` files.

**Comment by addyosmani** [comment]
Passing the buck to @sindresorhus for a second review of latest :)

**Comment by sindresorhus** [comment]
> and gitignore'd .DS_Store files.

That should be in your own global gitignore. It's not project specific.

**Comment by sindresorhus** [comment]
Looks good! Thank you @chrisdwheatley :)

**Pull request merged**